app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -108,3 +108,84 @@ legend p {
     width: 100%;
   }
 }
+
+.select2-choice {
+  abbr {
+    background: url("select2.png") right top no-repeat;
+  }
+
+  .select2-arrow {
+    b {
+      background: url("select2.png") no-repeat 0 1px
+    }
+  }
+}
+
+.navbar-brand {
+  background: url("govuk_admin_template/header-crown.png") 0 .67em no-repeat;
+}
+
+// stylelint-disable function-linear-gradient-no-nonstandard-direction 
```

Comment:
Should we be re-enabling the checks after (or using `stylelint-disable-line` or `stylelint-disable-next-line` on each individual line)?

(also applies later in this file)